### PR TITLE
Adds freeform and purge modules to Tramstation

### PR DESCRIPTION
**Pull request by SpaceSmithers**
## About The Pull Request
Tramstation's AI upload was missing the freeform and purge modules, which doesn't seem right according to #67915. This re-adds these modules and enforces upload standards.

## Why It's Good For The Game
Freeform and purge modules are present in every other station's AI upload.
ai_module.dm states the following via a comment: `AI uploads have the ai_module/reset , ai_module/supplied/freeform , ai_module/reset/purge , and ai_module/core/full/asimov directly mapped in`
In other words, Tramstation _not_ having these modules appears erroneous.

## Changelog

:cl:
fix: Tramstation's AI Upload is now equipped with the proper modules.
/:cl:

### CODE REVIEW

**Comment by SpaceSmithers** [comment]
Turns out it was also missing the purge module. Updated the info accordingly.

**Review by san7890 — APPROVED** [review]
It was likely accidentally regressed.
